perfkitbenchmarker/benchmark_spec.py

```diff
@@ -159,6 +171,25 @@ def _GetOsTypeForGroup(self, group_name):
       return group_spec[OS_TYPE]
     return FLAGS.os_type
 
+  def _CheckBenchmarkSupport(self, cloud):
+    """ Throw an exception if the benchmark isn't supported."""
+
+    if FLAGS.benchmark_compatibility_checking is SKIP_CHECK:
+      return
+
+    provider_info_class = provider_info.GetProviderInfoClass(cloud.lower())
```

Comment:
I think I'd prefer not using lower() and just using "cloud". This would match GetVmSpecClass and GetVmClass. The only reason providers.LoadProvider is called with cloud.lower() is so that the name will match the name of the provider package which is always lowercased (gcp/aws etc).